lib/lib_rss.php

```diff
@@ -390,6 +390,10 @@ function customSimplePie(array $attributes = [], array $curl_options = []): \Sim
 	if (is_array($force)) {
 		$https_domains = array_merge($https_domains, $force);
 	}
+
+	// Filter lines containing a # / ; as comments
+	$https_domains = array_filter($https_domains, fn(string $v) => !preg_match('/\\s*[\/#;]/', $v));
```

Comment:
It somehow seems both a bit overcomplicated and too simple at the same time? :-)

When going for complicated I'd expect something along these lines:
```php
$v = preg_replace('/\s*[\/#;].*$/', '', $v);
return !empty(trim($v));
```

Reply:
What case are you worried about?
`/ # ;` are all invalid characters in domain names anyway

Reply:
> What case are you worried about?

Reading the code and wondering why it does something imo unexpected wrt comments.

Of course if you meant it with a ^ then it's not doing anything particularly unexpected.

Reply:
In other words, I think a comment means "everything following character x" or "a line starting with x" (roughly the same thing anyway), not "this line randomly happens to contain character x".

Reply:
Indeed. The whitespace is only because I wanted to anchor the regex at the start (which I had initially forgotten), so it looks like that now: `/^\\s*[\/#;]/`
But if you do not like it, it would also be fine for me to just search for `/#;`

Reply:
Doesn't matter much really, as long as that `^` is there. It's just a slightly weird in between situation to me, where it doesn't support end of line comments yet has logic to support leading whitespace. :-)

Reply:
Reworked with a more proper support for comments, also end-of-line comments

Reply:
Sorry, haha. ^_^